test/Test/BaseDAO/Doc.hs

```diff
@@ -0,0 +1,17 @@
+-- SPDX-FileCopyrightText: 2020 TQ Tezos
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+module Test.BaseDAO.Doc
+  ( test_baseDAO_documentation
+  ) where
+
+import Prelude
+import Test.Tasty (TestTree, testGroup)
+
+import Lorentz (buildLorentzDoc, cCode)
+import Lorentz.Test
+import Lorentz.Contracts.BaseDAO (baseDaoContract, defaultConfig)
+
+test_baseDAO_documentation :: TestTree
+test_baseDAO_documentation = testGroup "baseDAO documentation" $
+  runDocTests testLorentzDoc $ buildLorentzDoc $ cCode $ baseDaoContract (defaultConfig @())
```

Comment:
Recently `GameDAO` was merged (see #41), we should test its documentation as well.

Reply:
Am I understanding it right, that here only `gameDaoProposalCheck`, `gameDaoRejectedProposalReturnValue`, `decisionLambda` and `gameDaoContract` should be commented?

Reply:
How do you want to comment them? If you mean haddock comments, they are already commented.

You need to add `runDocTests testLorentzDoc $ buildLorentzDoc $ cCode gameDaoContract` somewhere here. Maybe this test will pass immediately, but if it fails, you'll need to fix it.